package.json

```diff
@@ -21,6 +21,7 @@
   "devDependencies": {
     "@changesets/cli": "^2.27.10",
     "@eslint/js": "^9.22.0",
+    "@upstash/redis": "^1.34.4",
```

Comment:
Added this to resolve a dependency issue that was causing multiple versions of Wagmi to be installed. This will not be an issue in v1 since we're going to move wagmi/viem to peer dependencies in OnchainKit